guide/getting-started/software.md

```diff
@@ -152,10 +148,14 @@ Decentralized exchanges fall in between these categories, focusing on occasional
    layout = "float-right-desktop"
 %}
 
-A [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer running a Bitcoin software. Nodes verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. Some wallets have built-in nodes, and some node software comes packaged with wallet functionality. However, the minimum implementation of a node is syncing data and allowing other software a user has to access it.
+A Bitcoin [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer running software, like [Bitcoin Core]({{ '/guide/glossary/#bitcoin-core-client'}}) connecting in to the Bitcoin network. Nodes verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. Some wallets have built-in nodes, and some node software comes packaged with wallet functionality. However, the minimum implementation of a Bitcoin node is syncing data and allowing other software a user has to access it.
+
+A Lightning node is a device running software connecting it to the Lightning network, which extends Bitcoin with payment channels to improve transaction speed and lower costs. It is becoming widely adopted and accepted as the preferred way to scale Bitcoin.
+It’s common to use node management software. This simplifies the setup, management and monitoring of nodes by providing graphical user interfaces to interact with the lower level node software instead of CLIs.
```

Comment:
I would dedicate a whole section to this and go into the nuances of how one can operate their node (personal, merchant, routing node for example). Whole lot of detail here that is missing imo.

Reply:
None of the other sections go into that much detail about HOW to run the software.
If others agree we should cover the details here, then lets review if we should have a separate 'Node management' section.

Reply:
For now I separated the node management info into its own paragraph in f593a43.

Reply:
Whole section as in a separate guide section or page? We have the node page in the glossary already, and I agree with Daniel to keep this overview page light. People can click through to read more.

Reply:
Lightning node software and Lightning node management software are two distinct categories. Not saying go into great detail but I think it's relevant enough to have its own section on this page. Later on we could have a dedicated page that goes into the different node operation modes in detail similar to how we have a node page in the glossary.